lib/gooddata/lcm/actions/provision_clients.rb

```diff
@@ -93,15 +109,20 @@ def call(params)
                 end
 
                 params.gdc_logger.debug "Deleted clients: #{deleted_client_ids.join(', ')}"
-                raise error_message unless error_message['TooManyProjectsCreatedException'] || error_message['Max number registered projects']
+                unless error_message['TooManyProjectsCreatedException'] || error_message['Max number registered projects']
+                  raise error_message unless collect_synced_status
```

Comment:
If should be ```raise error_message unless continue_on_error``` .
In the case failed to provisioning a client because of TooManyProjectsCreatedException, we should continue to sync the other clients (provisioned successful) to not let these clients in invalid status.

Reply:
Done update code